.circleci/config.yml

```diff
@@ -187,17 +187,19 @@ jobs:
           no_output_timeout: 60m
           command: |
             pids=()
+
             npx nx-cloud record -- nx format:check --base=$NX_BASE --head=$NX_HEAD &
             pids+=($!)
 
-            npx nx run-many -t check-imports check-commit check-lock-files root-lint depcheck documentation --parallel=1 --no-dte &
+            npx nx run-many -t check-imports check-commit check-lock-files lint depcheck documentation --parallel=1 --no-dte &
```

Comment:
is this `lint` doing the same as `nx affected --target=lint` below?

Reply:
different, it's the `workspace-lint` target